store.html

```diff
@@ -0,0 +1,3160 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/store.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <!--body starts here-->
+  <body>
+    <!--header starts here-->
+    <header class="header">
+      <img src="img/karma-logo.svg" alt="karma-logo" class="navbar-logo" />
+
+      <nav>
+        <ul class="navbar-list">
+          <li id="navbar-button"><a href="#">Meet Karma</a></li>
+          <li id="navbar-button"><a href="#">How It Works</a></li>
+          <li id="navbar-button"><a href="#">Store</a></li>
+          <li id="navbar-button"><a href="#">Blog</a></li>
+          <li id="navbar-button"><a href="#">Help</a></li>
+          <li id="navbar-button"><a href="#">Login</a></li>
+        </ul>
+      </nav>
+    </header>
+    <!--main starts here-->
+    <main>
+      <section class="main-section">
+        <div class="form">
+          <h1>Order your Karma wifi<br />device today!</h1>
+          <div class="name">
+            <div class="field">
+              <label for="first">First name</label>
+              <input required type="text" name="first-name" />
+            </div>
+            <div class="field">
+              <label for="last">Last name</label>
+              <input required type="text" name="last-name" />
+            </div>
+          </div>
+          <div class="address">
+            <div class="field-2">
+              <label for="address">Address</label>
+              <input required type="text" name="address1" />
+            </div>
+            <div class="field-2">
+              <label for="address">Address 2</label>
+              <input required type="text" name="address2" />
+            </div>
+          </div>
+          <div class="city">
+            <div class="field-3">
+              <label for="city">City</label>
+              <!-- England-->
+              <select id="city" name="city">
```

Comment:
At the moment this is the width of its largest option. You could use CSS to set the `select` to be 100% of the width of its parent element.